### PR TITLE
Backoffice on-going

### PR DESCRIPTION
**Pull request by Omri-Levy**
- refactor(backoffice-v2): moved aml cells to a re-usable hook
- refactor(backoffice-v2): added an on-going blocks variant
- feat(backoffice-v2): added an actions variant for on-going
- feat(backoffice-v2): added temporary mocks

### Description
Elaborate on the subject, motivation, and context.

### Related issues
 * Provide a link to each related issue.

### Breaking changes
 * Describe the breaking changes that this pull request introduces.

### How these changes were tested
 * Describe the tests that you ran to verify your changes, including devices, operating systems, browsers and versions.

### Examples and references
 * Links, screenshots, and other resources related to this change.

### Checklist
- [] I have read the [contribution guidelines](CONTRIBUTING.md) of this project
- [] I have read the [style guidelines](STYLE_GUIDE.md) of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [] My changes generate no new warnings and errors
- [] New and existing tests pass locally with my changes

### CODE REVIEW

File: apps/backoffice-v2/src/domains/workflows/fetchers.ts [hunk 1]
```diff
@@ -141,6 +141,55 @@ export const fetchWorkflowById = async ({
             ...data.context?.pluginsOutput?.website_monitoring,
             data: deepCamelKeys(data.context?.pluginsOutput?.website_monitoring?.data ?? {}),
           },
+          kyc_session: {
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/domains/workflows/fetchers.ts]: Mocks to remove post integration.

File: apps/backoffice-v2/src/lib/blocks/components/NoBlocks/NoBlocks.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { NoTasksSvg } from '@/common/components/atoms/icons';
 
 export const NoBlocks = () => {
   return (
-    <div className="mb-72 flex items-center justify-center border-l-[1px] p-4">
+    <div className="flex items-center justify-center border-l-[1px] p-4 pb-72">
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/lib/blocks/components/NoBlocks/NoBlocks.tsx]: Fixes border not expanding the entire view.

File: apps/backoffice-v2/src/lib/blocks/components/AmlBlock/hooks/useAmlBlock/useAmlBlock.tsx [hunk 3]
```diff
@@ -0,0 +1,377 @@
+import { createBlocksTyped } from '@/lib/blocks/create-blocks-typed/create-blocks-typed';
+import * as React from 'react';
+import { ComponentProps, useMemo } from 'react';
+import { Badge } from '@ballerine/ui';
+import { WarningFilledSvg } from '@/common/components/atoms/icons';
+import { buttonVariants } from '@/common/components/atoms/Button/Button';
+import { amlAdapter } from '@/lib/blocks/components/AmlBlock/utils/aml-adapter';
+import { safeEvery } from '@ballerine/common';
+
+export const useAmlBlock = ({
```
Comment (Omri-Levy) [inline apps/backoffice-v2/src/lib/blocks/components/AmlBlock/hooks/useAmlBlock/useAmlBlock.tsx]: Abstracted existing code from `useKycBlock`.

**Review by tomer-shvadron — APPROVED** [review]

**Pull request merged**